docs/dev/relay_add_op.rst

```diff
@@ -139,6 +139,50 @@ before producing the call node:
         tup = Tuple(list(args))
         return _make.concat(tup)
 
+Gradient Operators
+------------------
+
+Gradient operators are important for writing differentiable programs in
+Relay. Without them, we couldn't train our models! Adding a gradient operator
+is slightly different from adding a normal operator, in that you only need to
+touch Python code. A collection of gradient operators can be found in
+``python/tvm/relay/op/_tensor_grad.py``. A good example is the gradient for
+``multiply``:
+
+.. code:: python
+
+    @register_gradient("multiply")
+    def multiply_grad(orig, grad):
+        """Returns [grad * y, grad * x]"""
+        x, y = orig.args
+        return [collapse_sum_like(grad * y, x),
+                collapse_sum_like(grad * x, y)]
+
+The inputs here are the original operator and a gradient to accumulate into.
+What we return is a list, where the 0th index is the derivative of the
+multiply operator with respect to the first input, and the 1st index is the
+derivative with respect to the second input. In general, the gradient will
+return a list with as many elements as there are inputs to the base operator.
+
+Before we further analyze this definition, first we should recall the partial
+derivatives for multiplication. Given a function f(x, y) = x * y, we have
+that df/dx = y and df/dy = x. The definition above looks similar to the math
+definitions, but there are some subtle differences, which we describe below.
+
+We're not just interested in how to compute the gradient of this function.
+We're interested in composing this gradient with other gradients, so we can
+accumulate the gradient across an entire program. This is where the ``grad *

```

Comment:
We may want a different choice of variable names because `grad * x` could be misinterpreted as ∇ * x (div). Also I think you should use :code: in front of the backticks

Reply:
Are you sure?  I've looked at the source for some of the docs, and all they use are double backticks for code formatting.

Reply:
Also, I'm not sure what you mean when you say "∇ * x (div)".

Reply:
https://en.wikipedia.org/wiki/Divergence

I don't know if other docs use something different (some kind of styling) but I have stuck to using `:code:` for Relay docs and found when I built docs that backticks without it ended up italicized, not with typewriter font.